app/uk/gov/hmrc/tai/connectors/NpsConnector.scala

```diff
@@ -94,8 +103,8 @@ class NpsConnector @Inject()(
     apiType: APITypes = APITypes.NpsIabdUpdateEstPayAutoAPI)(implicit hc: HeaderCarrier): Future[HttpResponse] =
     if (updateAmounts.nonEmpty) {
       val postUrl = npsPathUrl(nino, s"iabds/$year/employment/$iabdType")
-      postToNps[List[IabdUpdateAmount]](postUrl, apiType, updateAmounts)(
-        extraNpsHeaders(hc, version, sessionOrUUID),
+      postToNps[List[IabdUpdateAmount]](postUrl, apiType, updateAmounts, extraNpsHeaders(hc, version, sessionOrUUID))(
+        hc,
```

Comment:
can delete the implicits here, we no longer need to specify them

Reply:
I need to state formats.formatList, so I left that in and changed hc to implicitly